www/admin/artworks/get.php

```diff
@@ -6,7 +6,7 @@
 try{
 	$artwork = Artwork::Get($artworkId);
 }
-catch(Exceptions\SeException $ex){
+catch(Exceptions\SeException){
```

Comment:
```suggestion
catch(Exceptions\SeException $_){
```

I believe catching exceptions without assigning them to variables is only possible in PHP 8+. I'm not sure what `standardebooks.org` is actually running on but the README states 7+ and the included Dockerfile uses 7.4, so we should probably stick to syntax that is valid for PHP 7.